documentation/plain_rules.md

```diff
@@ -158,8 +158,9 @@ This substitution can be used in the following fields:
 Attribute value of `update` action and template of `post` action are expanded to numerical, boolean or JSON stringyfied
 values instead of string values when is possible. For example, if we have `{"a": "${x}"}`:
 
-* If the value of attribute `x` is `42` then it will expand do `{"a": 42}` and not to `{"a": "42"}`
-* If the value of attribute `x` is `{"hello": "world"}` then it will expand to `{"a": "{\"hello\":\"world\"}"}` (expand to native JSON, i.e. `{"a": {"hello": "world"}}`, is not supported)
+-   If the value of attribute `x` is `42` then it will expand do `{"a": 42}` and not to `{"a": "42"}`
+-   If the value of attribute `x` is `{"hello": "world"}` then it will expand to `{"a": "{\"hello\":\"world\"}"}`
+    (expand to native JSON, i.e. `{"a": {"hello": "world"}}`, is not supported)
```

Comment:
It seems my editor detects some invalid format and has fixed it

Reply:
NTC